tests/test_updatediff.py

```diff
@@ -649,6 +651,8 @@ def test_file_removed(tmp_path_factory: pytest.TempPathFactory) -> None:
         git("tag", "2")
     # Subproject updates
     with local.cwd(dst):
+        # git ignore a file
+        Path(".git/info/exclude").write_text("toignore.txt")
```

Comment:
Users don't usually do this. It's more common:

```suggestion
        Path(".gitignore").write_text("toignore.txt")
        git("commit", "-Am" "ignore file")
```

You can test both scenarios too by ignoring 2 files using those 2 different methods.